docs/source/tutorials/token_classification/utils.py

```diff
@@ -0,0 +1,242 @@
+            train.close() 
+            test.close() 
+        
+    indicies = [[index for index in indicies[i::k]] for i in range(k)] 
+    return indicies 
+
+def modified(given_words, sentence_tokens): 
+    for word, token in zip(given_words, sentence_tokens): 
+        if ''.join(word) != ''.join(token): 
+            return True  
+    return False 
+
+def process_token(token, replace=[('#', '')]): 
+    for old, new in replace: 
+        token = token.replace(old, new) 
+    return token 
+
+def mapping(entities, maps): 
+    f = lambda x: maps[x] 
+    return list(map(f, entities)) 
+
+def merge_probs(probs, maps): 
+    old_classes = probs.shape[1] 
+    probs_merged = np.zeros([len(probs), np.max(maps)+1]) 
+    
+    for i in range(old_classes): 
+        if maps[i] >= 0: 
+            probs_merged[:, maps[i]] += probs[:, i] 
+    if -1 in maps: 
+        row_sums = probs_merged.sum(axis=1) 
+        probs_merged /= row_sums[:, np.newaxis] 
+    return probs_merged 
+
+def get_probs(sentence, pipe, maps=None): 
+    def softmax(logit): 
+        return np.exp(logit) / np.sum(np.exp(logit)) 
+    
+    forward = pipe.forward(pipe.preprocess(sentence)) 
+    logits = forward['logits'][0].numpy() 
+    probs = np.array([softmax(logit) for logit in logits]) 
+    probs = probs[1:-1] 
+    
+    if not maps: 
+        return probs 
+    return merge_pros(probs, maps) 
+
+def get_pred_probs_and_labels(scores, tokens, given_token, given_label, weighted=False): 
+    i, j = 0, 0 
+    pred_probs, labels = [], [] 
+    for token, label in zip(given_token, given_label): 
+        i_new, j_new = i, j 
+        acc = 0 
+        
+        weights = []         
+        while acc != len(token): 
+            token_len = len(tokens[i_new][j_new:]) 
+            remain = len(token) - acc 
+            weights.append(min(remain, token_len)) 
+            if token_len > remain: 
+                acc += remain 
+                j_new += remain 
+            else: 
+                acc += token_len 
+                i_new += 1 
+                j_new = 0 
+        
+        if i != i_new: 
+            probs = np.average(scores[i:i_new], axis=0, weights=weights if weighted else None) 
+        else: 
+            probs = scores[i] 
+        i, j = i_new, j_new 
+        
+        pred_probs.append(probs) 
+        labels.append(label)
+        
+    return np.array(pred_probs), labels 
+
+def to_dict(nl): 
+    return {str(i): l for i, l in enumerate(nl)} 
+
+def read_npz(filepath): 
+    data = dict(np.load(filepath)) 
+    data = [data[str(i)] for i in range(len(data))] 
+    return data 
+
+def color_sentence(sentence, word): 
+    start_idx = sentence.index(word) 
+    before, after = sentence[:start_idx], sentence[start_idx + len(word):]
+    return '%s%s%s' % (before, colored(word, 'red'), after) 
+
+
+def show_issues(issues, labels, pred_probs, given_words, sentences, 
```

Comment:
Once we move this function to token_classification/summary.py, let's have it be next to the function currently named `show_sentence_issues` since these are highly related functions.